.prettierignore

```diff
@@ -15,4 +15,4 @@ CONTRIBUTING.md
 
 # npm files
 package.json
-
+/widget
```

Comment:
Doesn't look like npm file)

Reply:
agree) that folder was duplicated, removed